services/ui-src/src/page/OneMACForm.tsx

```diff
@@ -523,10 +532,20 @@ const OneMACForm: React.FC<{ formConfig: OneMACFormConfig }> = ({
             disabled={isSubmitting}
             fieldClassName="summary-field"
             multiline
-            onChange={handleInputChange}
+            onChange={(e) => {
+              handleInputChange(e);
+              setIsTyping(true);
+            }}
             value={oneMacFormData.additionalInformation}
             maxLength={config.MAX_ADDITIONAL_INFO_LENGTH}
+            aria-describedby="character-count"
+            aria-live="off"
+            aria-multiline={true}
+            aria-busy={isTyping}
           ></TextField>
+          <span tabIndex={0} aria-label="character-count" aria-live="polite">
+            {oneMacFormData.additionalInformation.length}/4000
```

Comment:
Would changing this to {4000-oneMacFormData.additionalInformation.length} remaining of 4000 be better?  I know the Figmas have the 0/4000, but AC does specify the remaining, not the used...